contracts/ERC5018.sol

```diff
@@ -7,6 +7,8 @@ import "./BlobStorageManager.sol";
 
 contract ERC5018 is IERC5018, LargeStorageManager, BlobStorageManager {
 
+    string public constant version = "1.0.0";
```

Comment:
Here is the Semver coding style used in the OP repository that we can follow. You can find more examples in the OP Stack smart contract [code](https://github.com/ethereum-optimism/optimism/blob/ebdae42966481bceda3decc272e60197d47c9fab/packages/contracts-bedrock/src/L1/OptimismPortal.sol#L132)

Reply:
We also need to inherit from [ISemver](https://github.com/ethereum-optimism/optimism/blob/ebdae42966481bceda3decc272e60197d47c9fab/packages/contracts-bedrock/src/L1/OptimismPortal.sol#L27)